ramsete-controller/robot.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+import wpilib
+import wpimath.controller
+import wpimath.filter
+import wpimath.geometry
+import wpimath.trajectory
+import wpimath.units
+
+from drivetrain import Drivetrain
+
+
+class MyRobot(wpilib.TimedRobot):
+    """
+    Uses the CameraServer class to automatically capture video from a USB webcam and send it to the
```

Comment:
wat

Reply:
What? Doesn't everyone use CameraServer in their ramsete implementation...? _sigh_... copy/paste got me again...